cli/tests/test_util_command.rs

```diff
@@ -43,7 +43,15 @@ fn test_gc_args() {
     let test_env = TestEnvironment::default();
     // Use the local backend because GitBackend::gc() depends on the git CLI.
```

Comment:
We already depend on the git CLI for push and pull tests, so I think it should be okay to update this test case to subprocess to `git` instead

Reply:
Naively replacing this with "git init" works. I don't understand the implications, but I'll add it in.

Reply:
Let me know if I can put something more meaningful in the commit description there.